molbe/_opt.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, pot, Fobjs, Nocc, enuc,solver='MP2', ecore=0.,
                  only_chem=False, hf_veff = None,
                  hci_pt=False,hci_cutoff=0.001, ci_coeff_cutoff = None, select_cutoff=None,
                  max_space=500, conv_tol = 1.e-6,relax_density = False,
-                 ebe_hf =0.):
+                 ebe_hf =0., scratch=None, **solver_kwargs):
```

Comment:
Wouldn't it make more sense to use existing `be_var.scratch`?

Reply:
It does indeed default to be_var.SCRATCH? All this kwarg accomplishes is to expose the fragment-level scratch directories to the user if desired.